### DISCUSSION TITLE
commits missing?

### DISCUSSION THREAD

**Discussion by whiteplainshistory** [community/community#21696] [New to GitHub]
<p>I have been committing to <a href="https://github.com/whiteplainshistory/whiteplainshistory.github.io" rel="nofollow noopener">https://github.com/whiteplainshistory/whiteplainshistory.github.io</a> happily for weeks.  Todays commit (from eclipse, as I have been doing) took some time but worked.  Yet when I visit github these recent changes are nowhere to be seen. It’s been a few hours now.  Please advise.</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/whiteplainshistory">@whiteplainshistory</a>,</p>
<p>Can you retry pushing to github from eclipse? There were problems earlier and the push might not have been successful.</p>

**Comment by whiteplainshistory** [comment]
<p>Seems better now.  Thx!</p>
